MIGRATION.md

```diff
@@ -0,0 +1,3 @@
+# Migration Guides
+
+To find migration guides for new versions of all packages see the `docs/migrations` directory.
```

Comment:
Can we add a link to the directory, please?

Reply:
Should it be a Github link or a local link?

Reply:
I think both should work...